evap/staff/tests/test_views.py

```diff
@@ -2229,16 +2230,18 @@ def test_questionnaire_with_answers_warning(self):
             '<label class="form-check-label badge bg-danger" for="id_contributions-1-questionnaires_0">', page
         )
 
-    @patch.dict(Evaluation.STATE_STR_CONVERSION, {Evaluation.State.PREPARED: "mock-translated-prepared"})
-    def test_state_change_log_translated(self):
-        page = self.app.get(self.url, user=self.manager)
-        self.assertNotIn("mock-translated-prepared", page)
+    @patch("django.utils.translation._trans", wraps=translation._trans)  # type: ignore[attr-defined]
+    def test_state_change_log_translated(self, trans):
+        trans.gettext = Mock()
+        trans.gettext.side_effect = lambda key: f"TRANSLATED-{key}"
 
         self.evaluation.ready_for_editors()
         self.evaluation.save()
 
-        page = self.app.get(self.url, user=self.manager)
-        self.assertIn("mock-translated-prepared", page)
+        response = self.app.get(self.url, user=self.manager)
+        self.assertContains(
+            response, "<li> TRANSLATED-state: TRANSLATED-new &#8594; TRANSLATED-prepared </li>", html=True
```

Comment:
I've observed `html=True` to be annoyingly slow in the past -- do we need it here? What's the average test runtime with and without the flag?

Reply:
comparing:
| Header | Header |
|--------|--------|
| `self.assertContains(response, "<li> TRANSLATED-state: TRANSLATED-new &#8594; TRANSLATED-prepared </li>", html=True)` | ~0.140s |
| `self.assertContains(response, "TRANSLATED-state: TRANSLATED-new &#8594; TRANSLATED-prepared", html=True)` | ~0.154s |
| `self.assertContains(response, "TRANSLATED-state: TRANSLATED-new &#8594; TRANSLATED-prepared")` | ~0.132s |

Yeah, I think without `html=True` it works just as well